operations/helm/charts/mimir-distributed/templates/metamonitoring/mixin-alerts.yaml

```diff
@@ -0,0 +1,20 @@
+{{- with .Values.metaMonitoring.serviceMonitor }}
+{{- if and .enabled .mimirAlerts }}
```

Comment:
originally this was validation against `serviceMonitor.enabled`. Are we just scrapping it in favor of `{{ - if .mimirAlerts }}`?

Reply:
yes, we'll document the need for servicemonitor